sim/rogue/dps_rogue/dps_rogue_test.go

```diff
@@ -11,76 +11,44 @@ func init() {
 	RegisterDpsRogue()
 }
 
-func TestCombat(t *testing.T) {
+func TestDaggers(t *testing.T) {
 	core.RunTestSuite(t, t.Name(), core.FullCharacterTestSuiteGenerator([]core.CharacterSuiteConfig{
 		{
 			Class:      proto.Class_ClassRogue,
-			Level:      25,
+			Level:      60,
+			Phase:      5,
 			Race:       proto.Race_RaceHuman,
 			OtherRaces: []proto.Race{proto.Race_RaceOrc},
 
-			Talents:     CombatDagger25Talents,
-			GearSet:     core.GetGearSet("../../../ui/rogue/gear_sets", "p1_combat"),
-			Rotation:    core.GetAplRotation("../../../ui/rogue/apls", "basic_strike_25"),
+			Talents:     CombatDaggers60Talents,
+			GearSet:     core.GetGearSet("../../../ui/rogue/gear_sets", "combat_daggers_60"),
+			Rotation:    core.GetAplRotation("../../../ui/rogue/apls", "combat_daggers_60"),
 			Buffs:       core.FullBuffsPhase1,
 			Consumes:    Phase1Consumes,
 			SpecOptions: core.SpecOptionsCombo{Label: "No Poisons", SpecOptions: DefaultCombatRogue},
 
-			ItemFilter:      ItemFilters,
-			EPReferenceStat: proto.Stat_StatAttackPower,
-			StatsToWeigh:    Stats,
-		},
-		{
-			Class:      proto.Class_ClassRogue,
-			Level:      40,
-			Race:       proto.Race_RaceHuman,
-			OtherRaces: []proto.Race{proto.Race_RaceOrc},
-
-			Talents:     CombatDagger40Talents,
-			GearSet:     core.GetGearSet("../../../ui/rogue/gear_sets", "p2_daggers"),
-			Rotation:    core.GetAplRotation("../../../ui/rogue/apls", "mutilate"),
-			Buffs:       core.FullBuffsPhase2,
-			Consumes:    Phase2Consumes,
-			SpecOptions: core.SpecOptionsCombo{Label: "No Poisons", SpecOptions: DefaultCombatRogue},
-
 			ItemFilter:      ItemFilters,
 			EPReferenceStat: proto.Stat_StatAttackPower,
 			StatsToWeigh:    Stats,
 		},
 	}))
 }
 
-func TestAssassination(t *testing.T) {
+func TestSwords(t *testing.T) {
 	core.RunTestSuite(t, t.Name(), core.FullCharacterTestSuiteGenerator([]core.CharacterSuiteConfig{
 		{
 			Class:      proto.Class_ClassRogue,
-			Level:      25,
+			Level:      60,
+			Phase:      5,
```

Comment:
Possibly my inexperience with the underlying data structure here, but do we think it's worth updating these phases to reflect the actual fresh Classic phases as to avoid confusion and improve accuracy for future resets and whatnot?

Reply:
We talked about this and decided we would get the baseline spec presets in place (#6), then update the phases.